WalletWasabi.Backend/Controllers/BlockchainController.cs

```diff
@@ -249,7 +250,8 @@ public async Task<IActionResult> BroadcastAsync([FromBody, Required] string hex)
 			catch (RPCException ex)
 			{
 				Logger.LogDebug(ex);
-				return BadRequest(ex.Message);
+				var spenders = Global.HostedServices.Get<MempoolMirror>().GetSpenderTransactions(transaction.Inputs.Select(x => x.PrevOut));
+				return BadRequest($"{ex.Message}:::{string.Join(":::", spenders.Select(x => x.ToHex()))}");
```

Comment:
Can you make this a bit more standard? Like a special exception or some convert/parse helper class? Here as well https://github.com/zkSNACKs/WalletWasabi/pull/5728/files#diff-f9d0ba1f58a9ea82f3fdcc5b7bfc13252c0bf00488d789e904d2c0e6d640ce10R111

Reply:
I am open for ideas. The limitation here is that it must be backwards compatible. So whatever you suggest, please go through the history of the code until the last non-backported release.

Reply:
My idea was to add a specific class to BadRequest because it can accept objects. As you mentioned compatibility is also a must-have, so my idea would fail in case of error here:
https://github.com/molnard/WalletWasabi/blob/47c91f3421cc236e390c5d2aa1f8a9d3e1ce1c1d/WalletWasabi/Tor/Http/Extensions/HttpResponseMessageExtensions.cs#L63

There is no further action we cannot replace this in backport, so only strings can be passed.